examples/integration_test.py

```diff
@@ -85,6 +88,18 @@ def error_cb(err):
     print('Error: %s' % err)
 
 
+def throttle_cb(t_report):
+    # validate argument type
+    assert isinstance(t_report, confluent_kafka.ThrottleEvent)
+
+    global throttled_requests
+    throttled_requests += 1
+
```

Comment:
I find it a bit unintuitive that this is called t_report since the throttle event is not called report anywhere else, maybe call it `throttle_event` instead?

Please add two more things:
 * check types of members (string, int, float)
 * Use the objects.__str__ in the print instead, e.g.: `print("ThrottleEvent: {}".format(t_report))"`